src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/samplers/RcaEnabledSampler.java

```diff
@@ -29,12 +30,12 @@ public void sample(SampleAggregator sampleCollector) {
     sampleCollector.updateStat(RcaRuntimeMetrics.RCA_ENABLED, "", isRcaEnabled());
   }
 
-  private int isRcaEnabled() {
+  @VisibleForTesting
+  int isRcaEnabled() {
```

Comment:
I would expect this to return a `boolean` :)

Reply:
Done